src/includes/getting-started-primer/android.mdx

```diff
@@ -10,16 +10,28 @@ The SDK builds a crash report that persists to disk and tries to send the report
 
 - The Native Development Kit (NDK), the set of tools that that allows you to use C and C++ code with Android, is packed with the SDK.
 - Events [enriched](/platforms/android/enriching-events/context/) with device data
-- Offline caching when a device is offline; we send a report once we receive another event
+- Offline caching when a device is offline; we send a report once the application is restarted
 - [Breadcrumbs automatically](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) captured for:
     - Android activity lifecycle events
     - Application lifecycle events (lifecycle of the application process)
     - System events (low battery, low storage space, airplane mode started, shutdown, changes of the configuration, and so forth)
     - App. component callbacks
+    - Android fragment lifecycle events with [Fragment Integration](/platforms/android/configuration/integrations/fragment/)
+    - OkHttp requests with [OkHttp Integration](/platforms/android/configuration/integrations/okhttp/)
+    - Apollo requests with [Apollo Integration](/platforms/android/performance/instrumentation/apollo/)
+    - Timber logs with [Timber Integration](/platforms/android/configuration/integrations/timber/)
 - [Release Health](/product/releases/health/) tracks crash free users and sessions
 - [Attachments](/platforms/android/enriching-events/attachments/) enrich your event by storing additional files, such as config or log files.
 - [User Feedback](/platforms/android/enriching-events/user-feedback/) provides the abiity to collect user information when an event occurs.
-- [Performance Monitoring](/product/performance/) creates transactions
+- [Performance Monitoring](/product/performance/) creates transactions for:
+    - Android activity transactions
+    - Android fragment spans with [Fragment Integration](/platforms/android/configuration/integrations/fragment/)
+    - Cold and warm app start spans and measurements with [Automatic Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#app-start-instrumentation)
+    - Slow and frozen frames measurements with [Automatic Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#slow-and-frozen-frames)
+    - OkHttp request spans with [OkHttp Integration](/platforms/android/configuration/integrations/okhttp/)
+    - SQLite and Room query spans with [Sentry Android Gradle Plugin](/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation)
```

Comment:
`m`: Maybe worth mentioning here that this is still in beta?

Reply:
its a link to the page, when somebody clicks on it, they will read the beta note